RELEASE_NOTES.txt

```diff
@@ -37,9 +37,10 @@ New Features:
 
 News Relevant to Package Maintainers
 ------------------------------------
-- Branch "master" uses GTK2, branch "gtk3" requires GTK+ >=3.16.
-- There are known bugs using GTK+ versions 3.20-3.22 where scrollbars may appear stuck (issue #3545) and where the Retinex tool's "Gain and Offset" panel may appear under the "Transmission" panel (issue #3525) until the user hovers the mouse cursor over a curve button. For this reason we recommend using GTK+ 3.16-3.18 if possible.
+- Branch "master" uses GTK2, branch "gtk3" requires GTK+ >=3.16. There are known bugs using GTK+ versions 3.20-3.22 where scrollbars may appear stuck (issue #3545) and where the Retinex tool's "Gain and Offset" panel may appear under the "Transmission" panel (issue #3525) until the user hovers the mouse cursor over a curve button. For this reason we recommend using GTK+ 3.16-3.18 if possible.
 - RawTherapee 5 requires GCC-4.9 or higher, or Clang.
+- Do not use -ffast-math
+- Use -o3
 - Use -DCMAKE_CXX_FLAGS="-std=c++11"
```

Comment:
@Beep6581 This can be dropped. As we found out, the `-std=gnu++11` in `CMakeLists.txt` takes precedence over the one supplied on the command line.